config/locales/en.yml

```diff
@@ -313,19 +313,25 @@ en:
       missing: "You must show the DLUHC privacy notice to the tenant before you can submit this log."
 
     scheme:
-      deactivation_date:
+      toggle_date:
         not_selected: "Select one of the options"
         invalid: "Enter a valid day, month and year"
         out_of_range: "The date must be on or after the %{date}"
+      reactivation:
+        before_deactivation: "This scheme was deactivated on %{date}. The reactivation date must be on or after deactivation date"
+      deactivation:
+        during_deactivated_period: "The scheme is already deactivated during this date, please enter a different date"
+
+
 
     location:
       toggle_date:
         not_selected: "Select one of the options"
         invalid: "Enter a valid day, month and year"
         out_of_range: "The date must be on or after the %{date}"
       reactivation:
-        before_deactivation: "This location was deactivated on %{date}\nThe reactivation date must be on or after deactivation date"
-      deactivation: 
+        before_deactivation: "This location was deactivated on %{date}. The reactivation date must be on or after deactivation date"
```

Comment:
The new line wasn't dislaying correctly so this is an alternative to avoid making a special case/adding multiple errors which seems a bit hacky.